themes/aframe/source/css/home.styl

```diff
@@ -50,8 +51,9 @@ ul
 
 .home__nav
   display flex
-  li
-    margin-right 10px
+  margin 30px $padding-med $padding-med 0
```

Comment:
Hehe.  I don't want to use another name for white spacing.  but $spacing-med  might be better?